index.html

```diff
@@ -12,13 +12,53 @@
     </head>
 
     <body>
-        <header>
-            <div>
-                <i class="icon-arrow-head"></i>
-                <span id="dummy-text-rem">with 1rem font size</span>
-                <span id="dummy-text-px">with 1px font size</span>
-            </div>
-        </header>
+        <!-- container is used to provide base margin  -->
+        <div class="container">
+            <header>
+                <nav class="navbar">
+                    <a href="/" class="navbar__logo">
```

Comment:
navbar__logo should go to the img tag
```suggestion
                    <a href="/" class="navbar__logo-wrapper">
```